x-pack/plugins/apm/public/components/app/ErrorGroupDetails/DetailView/index.tsx

```diff
@@ -281,7 +281,7 @@ export function TabContent({
         <Stacktrace stackframes={excStackframes} codeLanguage={codeLanguage} />
       );
     default:
-      const propData = error.context[currentTab.key] as any;
+      const propData = get(error, currentTab);
```

Comment:
We can't switch to idx here because the path is dynamic, is that right?

Reply:
Yes, `idx` was giving me more problems than help here so punting it for now. But I have some ideas how we can make all the properties statically typed.